packages/shared/src/hooks/useScrollTopClassName.ts

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+import { useScrollTopOffset } from './useScrollTopOffset';
+
+interface UseScrollTopStyleProps {
+  scrolledClassName?: string;
+  defaultClassName?: string;
+  scrollProperty?: 'scrollTop' | 'scrollY';
+  offset?: number;
+}
+
+export const useScrollTopClassName = ({
```

Comment:
We seem to depend on scroll for setting the background to either default or transparent, but curious why do we need this?

Reply:
The designs call for a transparent navbar. But when you start scrolling, the content goes below the navbar. Which looks ugly and makes the navbar less readable IMO. 

Try setting the navbar background to transparent !important on preview to see how it behaves if we do not have this code.